### DISCUSSION TITLE
발급 관련 URL 이슈

### DISCUSSION THREAD

**Discussion by etff** [CodeSoom/DDD-Kurly-Clone-Promotion#42] [General]
https://github.com/CodeSoom/DDD-Kurly-Clone-Promotion/issues/37#issue-1032525107

발급이 되려면 사용자
~~에게 발급해야 되는데 사용자 id는 어떻게 표현하면 좋을까요?

/coupons/issues/{userId}
필수 파라미터: keyword: string
으로 처리하면 좋을까요?

아니면
/coupons/issues로 두고
body로 모든 값을 받는 게 좋을까요?

**Comment by hannut91** [comment]
Url에 패스 파라미터로 userId를 받는다는 것은 사용자 아이디가 어떤 리소스에 대해서 명시를 해준다는 것인데, 여기에서는 우리가 쿠폰 이슈를 만들것이고, 그에 필요한 키워드 이름과 사용자 아이디를 받는 것이니, 저는 아래가 더 좋을 것 같습니다

**Comment by JamieShin0201** [comment]
저도 아래가 좋을 것 같습니다!
